src/app/utils/formValidationUtils.js

```diff
@@ -138,10 +123,12 @@ function validate(form, cb) {
       validate: () => true,
       errorMsg: '',
     },
+    // optional
```

Comment:
If these are optional, you can do `validate: () => true` so it doesn't actually check the value of the input.